dateparser/timezones.py

```diff
@@ -10,9 +10,11 @@
         'replace':
             [
                 (r'(?:UTC|GMT)\\(\+|\-)0(\d):00', r'(?:UTC|GMT)\\\1\2'),  # UTC+n, UTC-n, GMT+n, GMT-n
-                (r'(?:UTC|GMT)\\(\+|\-)0(\d):(\d{2})', r'(?:UTC|GMT)\\\1\2:\3'),  # UTC+n:mm, UTC-n:mm,GMT+n:mm, GMT-n:mm
+                (r'(?:UTC|GMT)\\(\+|\-)0(\d):(\d{2})', r'(?:UTC|GMT)\\\1\2:\3'),
+                # UTC+n:mm, UTC-n:mm, GMT+n:mm, GMT-n:mm
```

Comment:
I believe inline comments are usually moved to the line above, not below. You can append a `:` at the end of the comment if you think it makes things more clear.